src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -86,4 +127,10 @@ interface IEigenPodManager is IPausable {
     function slasher() external view returns(ISlasher);
 
     function hasPod(address podOwner) external view returns (bool);
+
+    /// @notice returns shares of provided podOwner
+    function getBeaconChainETHShares(address podOwner) external returns (uint256);
+
+    /// @notice returns canonical beaconChainETH strategy
```

Comment:
seems important to note that this is "virtual", i.e. this is not any actually existing contract